app/src/main/java/com/glia/exampleapp/MainFragment.java

```diff
@@ -480,9 +480,11 @@ private void showVisitorCode() {
     // For testing the integrated Visitor Code solution
     private void showVisitorCodeInADedicatedView() {
         VisitorCodeView visitorCodeView = GliaWidgets.getCallVisualizer().createVisitorCodeView(getContext());
-        CardView cv = containerView.findViewById(R.id.container);
-        cv.addView(visitorCodeView);
-        cv.setVisibility(View.VISIBLE);
+        if (visitorCodeView != null) {
+            CardView cv = containerView.findViewById(R.id.container);
+            cv.addView(visitorCodeView);
+            cv.setVisibility(View.VISIBLE);
+        }
```

Comment:
I think some info log could be useful here

Reply:
Why? This is the sample application and will not be used in debugging?

Reply:
Yes. That makes sense 👍